lib/modulesync/git.rb

```diff
@@ -27,9 +28,25 @@ def self.pull(git_base, name, branch, opts)
       end
     end
 
+    def self.bump(repo, m)
+      m.bump!
+      puts "Bumped to version #{m.version}"
+      repo.add('metadata.json')
+      repo.commit("Release version #{m.version}")
```

Comment:
Also problematic here, because the metadata version is different than what the commit indicates.